utilities/after_success.sh

```diff
@@ -34,9 +34,9 @@ if [ "${TRAVIS_JDK_VERSION}" == "oraclejdk7" ]; then
 
             cd ..
             utilities/update_docs_version.sh # Update version in READMEs
-            mvn clean deploy --settings ~/.m2/settings.xml -P sign-deploy
+            mvn clean deploy --settings ~/.m2/settings.xml -P sign-deploy,release
```

Comment:
The `sign-deploy` profile is defined in [`settings.xml`](https://github.com/GoogleCloudPlatform/gcloud-java/blob/travis/settings.xml#L31) and contains gpg signing configuration.

Yes I tested the command and gpg configuration is loaded correctly (deploy fails as I don't have any gpc key in the path specified in `settings.xml`). In fact, according to `mvn` docs:
```
-P,--activate-profiles <arg>           Comma-delimited list of profiles
                                       to activate
```